common/scheme/scheme.go

```diff
@@ -70,11 +72,16 @@ func ReadSchemeByEnv() (Scheme, bool) {
 
 // GetSchemeFromEnv allows the user to retrieve the scheme configuration looking by the ID set on an
 // environmental variable. If the scheme is not found, function will panic.
-func GetSchemeFromEnv() Scheme {
+func GetSchemeFromEnv() crypto.Scheme {
```

Comment:
Ideally this whole `scheme` package should go, but for now I'm just making it return a `crypto.Scheme`.

TODO.